docs/pages/installation/index.md

```diff
@@ -16,12 +16,21 @@ Getting Meshery up and running locally on a Docker-enabled system or in Kubernet
 
 Use the Meshery command line interface, [mesheryctl]({{ site.baseurl }}/guides/mesheryctl), to install and start Meshery. Begin with Meshery by installing its command line client: [mesheryctl]({{ site.baseurl }}/guides/mesheryctl). If you are on a MacOS or Linux system, you can download, install, and run to the management plane with the command shown in the figure.
 
+ <span style="font-size: 20px">`Install on Docker`</span>
  <pre class="codeblock-pre"><div class="codeblock">
  <div class="clipboardjs">
- $ curl -L https://git.io/meshery | bash - 
+ $ curl -L https://git.io/meshery | PLATFORM=docker bash - 
  </div></div>
  </pre>
+ <br>
 
+ <span style="font-size: 20px">`Install on Kubernetes`</span>
```

Comment:
@sayantan1413 ideally, we're avoiding inline style where possible. Did you try one of the smaller headings or bold here? How did they look?

Reply:
Okay no problem I am changing the **span** to **h6**  it looks almost similar.

Reply:
Thanks. The backticks are unnecessary, too. And should be reserved for code / cli only.